### ISSUE TITLE
[Frozen TensorFlow Parser] convert tensorflow to IR

### ISSUE THREAD

**Issue by zhengduoru** [Microsoft/MMdnn#121]
Platform (like ubuntu 16.04/win10):
ubuntu 16.04
Python version:
2.7
Source framework with version (like Tensorflow 1.4.1 with GPU):
Tensorflow 1.6.0

zdr@arcsh1080:~/github/models/research/deeplab/datasets/pascal_voc_seg/exp/train_on_trainval_set/train$ python3 -m mmdnn.conversion._script.convertToIR -f tensorflow -n model.ckpt-0.meta -w model.ckpt-0.data-00000-of-00001 --dstNode ResizeBilinear_3 -o converted

/usr/lib/python3.4/importlib/_bootstrap.py:321: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
  return f(*args, **kwds)
Parse file [model.ckpt-0.meta] with binary format successfully.
Tensorflow model file [model.ckpt-0.meta] loaded successfully.
2018-03-20 16:25:41.159577: W tensorflow/core/util/tensor_slice_reader.cc:95] Could not open ./model.ckpt-0.data-00000-of-00001: Data loss: not an sstable (bad magic number): perhaps your file is in a different file format and you need to use a different restore operator?
Traceback (most recent call last):
  File "/usr/lib/python3.4/runpy.py", line 170, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.4/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/usr/local/lib/python3.4/dist-packages/mmdnn/conversion/_script/convertToIR.py", line 158, in <module>
    _main()
  File "/usr/local/lib/python3.4/dist-packages/mmdnn/conversion/_script/convertToIR.py", line 153, in _main
    ret = _convert(args)
  File "/usr/local/lib/python3.4/dist-packages/mmdnn/conversion/_script/convertToIR.py", line 51, in _convert
    parser = TensorflowParser(args.network, args.weights, args.frozen_pb, args.dstNodeName)
  File "/usr/local/lib/python3.4/dist-packages/mmdnn/conversion/tensorflow/tensorflow_parser.py", line 189, in __init__
    self.ckpt_data = TensorflowParser._load_weights(checkpoint_file)
  File "/usr/local/lib/python3.4/dist-packages/mmdnn/conversion/tensorflow/tensorflow_parser.py", line 104, in _load_weights
    reader = tensorflow.train.NewCheckpointReader(model_weight_path)
  File "/usr/local/lib/python3.4/dist-packages/tensorflow/python/pywrap_tensorflow_internal.py", line 150, in NewCheckpointReader
    return CheckpointReader(compat.as_bytes(filepattern), status)
  File "/usr/local/lib/python3.4/dist-packages/tensorflow/python/framework/errors_impl.py", line 473, in __exit__
    c_api.TF_GetCode(self.status.status))
tensorflow.python.framework.errors_impl.DataLossError: Unable to open table file ./model.ckpt-0.data-00000-of-00001: Data loss: not an sstable (bad magic number): perhaps your file is in a different file format and you need to use a different restore operator?

The model was pretrained in tensorflow's deeplab.
I can't even convert it to IR.

**Comment by zhengduoru** [comment]
Tensorflow has not supported operator [Equal] with name [Equal].
My tensorflow is 1.6.0

**Comment by zhengduoru** [comment]
my model can be download from 
http://download.tensorflow.org/models/deeplabv3_pascal_train_aug_2018_01_04.tar.gz
I use the frozen_inference_model.pb in the tar

**Comment by kitstar** [comment]
Frozen TensorFlow graph is not formal supported yet. We will ping you if there is any progress.

**Comment by giticaniup** [comment]
@kitstar Hi, do you have a time plan to implement the feature  [Frozen TensorFlow Parser]? I use the current implement but have some bugs.

**Comment by kitstar** [comment]
Hi @giticaniup  could you provide the error model for us to test? Thanks.

**Comment by giticaniup** [comment]
@kitstar Sorry I can't upload the model file because of network limitation.
I tested with inpception_v3_2016_08_28_frozen.pb.tar.gz, that is ok. When I tested the quantitative version which generated by tool bazel-bin/tensorflow/tools/quantization/tools/quantize_graph, errors occured.
One of them is I can't read weight value for the convolution layer in function rename_QuantizedConv2D(tensorflow_frozenparser.py), and I have no  idea how to solve it.
Hope this information  is useful, I need your help.
